src/services

```diff
@@ -1122,4 +1122,26 @@ p({
         }
     }
   """
+})
+
+
+p({
+  "class":"foam.nanos.boot.NSpec",
+  "name":"localSettingDAO",
+  "lazy":false,
+  "serve":true,
+  "authenticate": false,
+  "serviceScript":"""
+    import foam.dao.DAO;
+    dao = x.get("localLocalSettingDAO");
+
+    if(dao == null)
+      dao = new foam.dao.NullDAO(x, foam.nanos.session.LocalSetting.getOwnClassInfo());
+    return new foam.dao.EasyDAO.Builder(x)
+      .setInnerDAO(dao)
+      .setAuthorize(false)
```

Comment:
I will fix setAuthorize later

Reply:
authorize false is fine, it's only in the logged in users context.